Resources/public/js/views/fields/ez-dateandtime-view.js

```diff
@@ -79,11 +79,22 @@ YUI.add('ez-dateandtime-view', function (Y) {
         _getDateObject: function () {
             var field = this.get('field');
 
-            if ( field && field.fieldValue ) {
+            if (this._isFieldEmpty()) {
+                return undefined;
+            } else {
```

Comment:
nitpick: no need for else
nitpick2: while at it you can update the doc of the method to indicate this method can return `undefined`